packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -649,6 +668,57 @@ contract Attestations is
     }
   }
 
+  /**
+   * @notice Update the approval status of allowing an attestation identifier 
+   * mapping to be transfered from the specified address to msg.sender
+   * @param source The current attesation address to be transfered
+   * @param status The approval status
+   */
+  function approveTransfer(address source, bool status) external returns (bool) {
```

Comment:
Posting this comment for posterity:
The current design requires that the transferee approve before the transferer transfers. Wdyt of the following?
```
mapping(address => mapping(bytes32 => bool)) transferApprovals;
function approveTransfer(bytes32 identifier, address from, address to, bool status) {
  bytes32 key = keccak(identifier, from, to);
  transferApprovals[msg.sender][key] = status;
  if (status && other party approved) {
     _transfer(identifier, from, to);
    clear approvals
  } 
}
```